anagrafica/migrations/0043_generate_new_slugs.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-01-27 19:34
+from __future__ import unicode_literals
+
+from django.core.exceptions import MultipleObjectsReturned
+from django.db import migrations
+
+
+def update_slug(apps, schema_editor):
+    Sede = apps.get_model('anagrafica', 'Sede')
+    for sede in Sede.objects.all():
```

Comment:
Mi vengono in mente un paio di modi per rendere questa operazione molto piu' efficiente (i.e. operazioni di gruppo SQL). Non e' un problema qui, e non c'e' bisogno di cambiarlo, pero' nel lungo termine evitiamo perche' quando bisogna rigenerare una installazione al volo (es. staging, locale, ecc.) bisogna riapplicare tutte le ultime migrazioni.